cli/src/services/config.ts

```diff
@@ -37,6 +38,18 @@ export default class ConfigFetcher {
       error('You must have at least 1 document set in your config');
     }
 
+    const branchName = this.getCurrentBranchName();
```

Comment:
I know this will be a bit less clean but I would prefer if `branchName` was only "computed" if `this.config.version?.branchVersionPrefix` is present. Now it will run the `git` command every time for every project

Reply:
Totally!